gmso/core/topology.py

```diff
@@ -1228,6 +1228,27 @@ def get_index(self, member):
 
         return index
 
+    def to_xml(self, filename, overwrite=False, backend="gmso"):
```

Comment:
Can we rename this? I think it's a bit confusing to have two functions to_xml(). I would suggest `write_forcefield`. 

Also, you can remove the backend argument. The only backend will be ffutils, as gmso file handling is being deprecated.